packages/xo-server/src/xapi/index.js

```diff
@@ -1658,11 +1658,11 @@ export default class Xapi extends XapiBase {
 
   async startVm(vmId, hostId, force) {
     try {
-      await this._startVm(
-        this.getObject(vmId),
-        hostId && this.getObject(hostId),
-        force
-      )
+      const vm = this.getObject(vmId)
+      await this._startVm(vm, hostId && this.getObject(hostId), force)
+      await this._updateObjectMapProperty(vm, 'other_config', {
+        start_time: vm.$metrics && vm.$metrics.start_time,
+      })
```

Comment:
Are you doing this because suspended/paused VMs don't have a `start_time`?

Reply:
it's for suspended VM.

Reply:
Ok but I don't think this is a good idea. If the VM is started from another client, it won't simply not work. It will show an erroneous start time! Also, if XenServer chose not to expose it for suspended VMs, there might be a good reason.